app/controllers/account/admin/issues_controller.rb

```diff
@@ -37,13 +40,28 @@ def destroy
 
       private
 
+      def facebook_posting(issue)
+        return if Rails.env.test?
+        page = prepare_facebook_page
+        page.feed!(message: "Адреса: #{issue.location}. \n \n
```

Comment:
yes + what you can do is to create instance method if "issue"

```
def fb_message
  {
     return you hash
  }
end 

+ add spec for this method, since you have return if Rails.env.test .... so you can verify that this code works
```